code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -178,6 +191,88 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 49;
 }
 
+type CosmwasmRouter = cosmwasm::ibc::Router<Runtime>;
+
+#[derive(Clone, Debug, Eq, PartialEq, Default)]
+pub struct Router {
+	pallet_ibc_ping: pallet_ibc_ping::IbcModule<Runtime>,
```

Comment:
We don't need this pallet here.

Reply:
How do I create the router then?

Reply:
You can replace it with the cosmwasm router